uint16 shoulderVal
uint16 elbowVal
uint16 baseVal
